haproxy-redundant-floatingip-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,506 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "defaultValue": "azureuser",
+      "metadata": {
+        "description": "Admin username"
+      }
+    },
+    "sshKeyData": {
+      "type": "string",
+      "metadata": {
+        "description": "SSH rsa public key file as a string."
+      }
+    },
+    "dnsNameforLBIP": {
+      "type": "string",
+      "defaultValue": "haproxylb",
+      "metadata": {
+        "description": "Unique DNS Name for Load Balancer Public IP"
+      }
+    },
+    "lbName": {
+      "type": "string",
+      "defaultValue": "myLB",
+      "metadata": {
+        "description": "Load Balancer name"
+      }
+    },
+    "vnetName": {
+      "type": "string",
+      "defaultValue": "myVNET",
+      "metadata": {
+        "description": "VNET name"
+      }
+    },
+    "haproxyVMNamePrefix": {
+      "type": "string",
+      "defaultValue": "haproxyvm-",
+      "metadata": {
+        "description": "Prefix to use for names of VMs under the load balancer"
+      }
+    },
+    "appVMNamePrefix": {
+      "type": "string",
+      "defaultValue": "appvm-",
+      "metadata": {
+        "description": "Prefix to use for names of application VMs"
+      }
+    },
+    "ubuntuOSVersion": {
+      "type": "string",
+      "defaultValue": "14.04.2-LTS",
+      "allowedValues": [
+        "12.04.5-LTS",
+        "14.04.2-LTS",
+        "15.04"
+      ],
+      "metadata": {
+        "description": "The Ubuntu version for the VM. This will pick a fully patched image of this given Ubuntu version. Allowed values: 12.04.5-LTS, 14.04.2-LTS, 15.04."
+      }
+    },
+    "vmSize": {
+      "type": "string",
+      "defaultValue": "Standard_D1",
+      "metadata": {
+        "description": "Size of the VM"
+      }
+    }
+  },
+  "variables": { 
+    "scriptsBaseUrl": "https://raw.githubusercontent.com/sivaedupuganti/azure-quickstart-templates/master/haproxy-redundant-floatingip-ubuntu/",
```

Comment:
Please fix this link as per https://github.com/Azure/azure-quickstart-templates#rawgithubusercontentcom-links